.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Publish Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USER }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASS }}
```

Comment:
I don't think we want to use a user and pass here, but have API token like so:

https://github.com/Unidata/MetPy/blob/main/.github/workflows/release.yml#L54

Reply:
Ahhh -- that makes sense. I don't have access to the repo's settings. Is the token already set there?

Reply:
I have set `PYPI_TOKEN` now

Reply:
Updated the file to use the token 🤞🏻